src/components/select/select-screen.js

```diff
@@ -33,15 +33,18 @@ export class Select extends Screen {
     create() {
         this.addBackgroundItems();
         createTitles(this);
-        const buttons = ["home", "pause", "previous", "next"];
+        const onePage = this.config.choices.length > this.config.columns * this.config.rows;
+        const pagingButtons = onePage ? ["previous", "next"] : [];
```

Comment:
if onePage is true, then it shows the previous and next buttons
think this should be renamed or the condition changed to check for one page instead of more than one page

Reply:
Oh yeah - that's totally backwards!